core/src/test/java/org/apache/calcite/test/SqlToRelTestBase.java

```diff
@@ -770,7 +795,32 @@ public void assertConvertsTo(
       assertConvertsTo(sql, plan, false);
     }
 
+    public void assertConvertsTo(String sql,
+                                 String plan,
+                                 boolean trim) {
+      assertConvertsTo(sql, plan, false, true);
+    }
+
     public void assertConvertsTo(
+            String sql,
+            String plan,
+            boolean trim,
+            boolean query) {
+      if (query) {
+        assertSqlConvertsTo(sql, plan, trim);
+      } else {
+        assertExprConvertsTo(sql);
+      }
+    }
+
+    private void assertExprConvertsTo(
+        String expr) {
```

Comment:
```java
private void assertExprConvertsTo(
        String expr,
        String plan) {
      String expr2 = getDiffRepos().expand("sql", expr);
      RexNode rex = convertExprToRex(expr2);
      assertNotNull(rex);
      // NOTE jvs 28-Mar-2006:  insert leading newline so
      // that plans come out nicely stacked instead of first
      // line immediately after CDATA start
      String actual = NL + rex.toString();
      diffRepos.assertEquals("plan", plan, actual);
    }
```

Reply:
Thanks @danny0405 , I had fixed my idea codeStyle. and I added a test case which is named testInWithConstantList in SqlToRelConverterTest . It can cover the code which I made change.